### PR TITLE
Add tile to toggle all animations (close #24)

### LINKED ISSUE #24: Add tile to toggle all animations
Add a tile to toggle the following - 
1. Window Animation Scale
2. Transition Animation Scale
3. Animator Duration Scale

This will be useful while switching between normal development and running Espresso tests.

### PR DESCRIPTION
**Pull request by devesh-shetty**

### CODE REVIEW

File: app/src/main/kotlin/xyz/mustafaali/devqstiles/MainActivity.kt [hunk 1]
```diff
@@ -51,4 +98,15 @@ class MainActivity : AppCompatActivity() {
...
     }
+
+    private fun getFeaturesList(): List<Feature> {
+        return listOf(
+                Feature("Toggle USB Debugging", "Enable/disable USB debugging from your notification drawer", R.drawable.ic_toggle_usb_debugging),
+                Feature("Keep Screen On", "Keep screen on when connected via USB, but turn it off when connected to a charger", R.drawable.ic_toggle_keep_screen_on),
+                Feature("Show Touches", "Show touch points when you touch the screen, ideal for demos", R.drawable.ic_toggle_show_taps),
+                Feature("Demo Mode", "Cleans up the status bar for those perfect screenshots", R.drawable.ic_toggle_demo_mode),
+                Feature("Change Animator Duration", "Change the default animator duration to easily debug animations", R.drawable.ic_animator_duration),
+                Feature("Toggle Animation Scale", "Toggle window animation scale, transition animation scale, and animator duration scale", R.drawable.ic_animation)
```
Comment (mustafa01ali) [inline app/src/main/kotlin/xyz/mustafaali/devqstiles/MainActivity.kt]: Update to `("Toggle all animations", "Enable/disable all animations with one click, perfect for running Espresso tests, ....)`

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -84,6 +85,18 @@
             </intent-filter>
         </service>
 
+        <service
+            android:name="xyz.mustafaali.devqstiles.service.ToggleAnimationService"
+            android:enabled="true"
+            android:exported="true"
+            android:icon="@drawable/ic_animation"
+            android:label="@string/label_toggle_animation_scale"
```
Comment (mustafa01ali) [inline app/src/main/AndroidManifest.xml]: Change this to "Toggle all animations"

**Review by mustafa01ali — CHANGES_REQUESTED** [review]
Looks good @devesh-shetty, just a couple of minor comments. Also, please sync your fork and rebase it off of development for a clean diff.

**Comment by devesh-shetty** [comment]
@mustafa01ali changes done!

**Review by mustafa01ali — APPROVED** [review]

**Comment by 1043465747** [comment]
Bink-dev  كم رصيدي ف البنك

**Pull request merged**